charts/ratify/values.yaml

```diff
@@ -1,8 +1,8 @@
 image:
-  repository: huishratifyimage.azurecr.io/deislabs/ratify
-  crdRepository: huishratifyimage.azurecr.io/deislabs/ratify-crds
-  tag: v1191
-  pullPolicy: Always
+  repository: ghcr.io/deislabs/ratify
+  crdRepository: ghcr.io/deislabs/ratify-crds
```

Comment:
I don't believe this exists yet and the plan was to update the release pipeline after this? Won't this cause the e2e tests to fail since there'a new CRD test in the bats file?

Reply:
actually never mind. I forgot we build the image everytime so we don't use the published image for testing.